plutus-core/plutus-core/src/PlutusCore/Evaluation/Error.hs

```diff
@@ -60,34 +60,34 @@ mtraverse makeClassyPrisms
     ]
 
 instance Bifunctor EvaluationError where
-    bimap f _ (OperationalEvaluationError err) = OperationalEvaluationError $ f err
-    bimap _ g (StructuralEvaluationError err)  = StructuralEvaluationError $ g err
+    bimap f _ (StructuralEvaluationError err)  = StructuralEvaluationError $ f err
+    bimap _ g (OperationalEvaluationError err) = OperationalEvaluationError $ g err
     {-# INLINE bimap #-}
 
 instance Bifoldable EvaluationError where
-    bifoldMap f _ (OperationalEvaluationError err) = f err
-    bifoldMap _ g (StructuralEvaluationError err)  = g err
+    bifoldMap f _ (StructuralEvaluationError err)  = f err
+    bifoldMap _ g (OperationalEvaluationError err) = g err
     {-# INLINE bifoldMap #-}
 
 instance Bitraversable EvaluationError where
-    bitraverse f _ (OperationalEvaluationError err) = OperationalEvaluationError <$> f err
-    bitraverse _ g (StructuralEvaluationError err)  = StructuralEvaluationError <$> g err
+    bitraverse f _ (StructuralEvaluationError err)  = StructuralEvaluationError <$> f err
+    bitraverse _ g (OperationalEvaluationError err) = OperationalEvaluationError <$> g err
```

Comment:
I've checked, we don't need to update any call sites, because there's only one and `f` and `g` are the same there:

```haskell
instance (AsUnliftingError operational, AsUnliftingError structural) =>
        AsUnliftingEvaluationError (EvaluationError structural operational) where
    _UnliftingEvaluationError = go . coerced where
        go =
            prism'
                (bimap
                    (review _UnliftingError)
                    (review _UnliftingError))
                (bitraverse
                    (reoption . matching _UnliftingError)
                    (reoption . matching _UnliftingError))
```